test/tests/searchTest.js

```diff
@@ -83,4 +83,57 @@ describe("Zotero.Search", function() {
 			assert.propertyVal(conditions[0], 'value', 'foo')
 		});
 	});
+
+	describe("#search()", function () {
+		let win;
+		let fooItem;
+		let foobarItem;
+
+		before(function* () {
+			// Hidden browser, which requires a browser window, needed for charset detection
+			// (until we figure out a better way)
+			win = yield loadBrowserWindow();
+			fooItem = yield importFileAttachment("search/foo.html");
+			foobarItem = yield importFileAttachment("search/foobar.html");
+		});
+
+		after(function () {
+			if (win) {
+				win.close();
+			}
+		});
+
+		it("should return results with full-text conditions", function* () {
+			let s = new Zotero.Search();
+			s.addCondition('fulltextWord', 'contains', 'foo');
+			let results = yield s.search();
+			assert.equal(results[0], fooItem.id);
+		});
```

Comment:
Is there a guarantee for the order? This should return both file attachments, no? Could use `assert.sameMembers` here